gateway/ln-gateway/src/lib.rs

```diff
@@ -1964,28 +2004,30 @@ impl Gateway {
             .federation_info(*federation_id, &mut dbtx)
             .await?;
 
-        let routing_fees: RoutingFees = if let Some(fees) = fed_config.routing_fees {
-            fees.into()
-        } else {
-            PaymentFee::SEND_FEE_DEFAULT.into()
-        };
+        let routing_fees = fed_config
+            .routing_fees
+            .unwrap_or(PaymentFee::TRANSACTION_FEE_DEFAULT);
+
+        let transaction_fees = fed_config
+            .transaction_fees
+            .unwrap_or(PaymentFee::TRANSACTION_FEE_DEFAULT);
```

Comment:
You can get this from the config directly instead of the FederationInfo.

Reply:
Fixed